python/lsst/pipe/tasks/quickFrameMeasurement.py

```diff
@@ -408,7 +455,7 @@ def _makeEmptyReturnStruct():
         result.medianXxYy = (np.nan, np.nan)
         return result
 
-    def run(self, exp, doDisplay=False):
+    def run(self, exp, *, donutDiameter=None, doDisplay=False):
```

Comment:
Am I missing where donutDiameter is documented below?

Reply:
Not _exactly_, I renamed the parameter from `donutSize` and did it everywhere in the code but just forgot the docstring 🤦